src/components/calcite-inline-editable/calcite-inline-editable.tsx

```diff
@@ -88,9 +88,8 @@ export class CalciteInlineEditable {
   }
 
   componentDidLoad() {
-    this.htmlInput = this.inputElement.querySelector("input");
     if (!this.editingEnabled) {
-      this.htmlInput.tabIndex = -1;
```

Comment:
I'm not sure why this existed but I think we should be passing a prop to the input instead of modifying tabindex. @eriklharper

Reply:
Good question.  Perhaps @alexabreu might have more context for that.  I have seen some issues where some kind of vdom weirdness requires direct setting of html attributes like that.  Sometimes just passing props through JSX doesn't work fully.

Reply:
It seems like if we need to set tabindex of an internal piece of the component there should be a prop like disabled or readOnly or something.